### PR TITLE
Move shared components into components/shared folder

### PR DESCRIPTION
**Pull request by mmafe**
# Summary of changes
- Move shared components into components/shared folder
- Allows us to proceed with adding custom blocks into separate components/block folder
- Use absolute paths for imports (by default it looks in the src directory)
- Move events fragment into Events component

## Frontend
- Shared components moved into components/shared folder
- Updated any references to those components
- Updated all imports so they use absolute paths

## Backend
- None

# Test Plan

- Visit test build: https://build-888d47a6-ff48-472d-9ece-c3468a366752.gtsb.io/
- Nothing should be different

### CODE REVIEW

**Comment by snschau** [comment]
should .wordpress-cache folder not be added to the main .gitignore file?

**Comment by snschau** [comment]
i get the following errors - on a build/develop  

warn Plugin gatsby-plugin-catch-links is not compatible with your gatsby version 3.14.6 - It requires gatsby@^4.0.0-next
warn Plugin gatsby-source-wordpress is not compatible with your gatsby version 3.14.6 - It requires gatsby@^4.0.0-zz-next.1
warn Plugin gatsby-plugin-catch-links is not compatible with your gatsby version 3.14.6 - It requires gatsby@^4.0.0-next
warn Plugin gatsby-source-wordpress is not compatible with your gatsby version 3.14.6 - It requires gatsby@^4.0.0-zz-next.1

Other than that it looks good.

**Comment by tqureshi-uog** [comment]
@snschau those are known issues with the gatsby-source-wordpress and catch-links plugins as per the earlier eventstest PR. They don't affect functionality and will disappear once we upgrade to gatsby4.

@mmafe I'm not sure if this is related to the PR, but when I go to this page on the PR build - https://build-4d1a9805-e0b9-4794-a1a4-1949b80fc9ce.gtsb.io/programs/bachelor-of-arts-general-fully-online - the two SVG icons don't show:

image

vs how it currently appears on https://ugconthub.gtsb.io/programs/bachelor-of-arts-general-fully-online:

image

**Comment by mmafe** [comment]
Good catch - will look into it

**Comment by mmafe** [comment]
So locally, the icons show up...and when I run shared-components on the MMtest  environment(Gatsby Cloud) using live-chug, the icons show up.

At first I thought it might have to do with the project dependencies being reinstalled on that PR build, but then I checked the PR build on TQtest, and that one shows the icons: https://build-4b65fb80-5b4c-485c-809b-081177004e6a.gtsb.io/programs/bachelor-of-arts-general-fully-online

In short, I'm stumped. The only place it seems to occur was on that specific build.

To see if I can recreate it, I'll put a test commit and see what happens on the PR build that gets generated from it.

**Comment by tqureshi-uog** [comment]
I wonder if it might have something to do with the fact the live Gatsby Cloud site is the only one that uses ASSET_PREFIX and PREFIX_PATHS (to add the https://cdn.uoguelph.ca URL in front of things like images). You could create a new branch from master, make some trivial formatting change, and create a PR from that to see if the same glitch occurs - it could be this was something that happened with every PR build against ugconthub and we only learned about it now (and if only affects PR builds, not the live site, we can safely ignore it)

**Comment by mmafe** [comment]
Note: At this point I'm ignoring the CHUG Test environment since that failed due to authentication issues with api.test.

After the commit...
- the UG Content Hub PR build is NOT showing the icons.
- The build on MMTest and TQ Test DO show the icons.
- The local build continues to show the icons.

I am completely stumped. Going to leave it until I have an answer. Will create a second PR to see what's going on.

**Review by snschau — APPROVED** [review]
looks good now

**Review by tqureshi-uog — APPROVED** [review]
